.gitattributes

```diff
@@ -0,0 +1,16 @@
+# Explicitly declare text files we want to always be normalized and converted on checkout.
```

Comment:
I followed this recommendation from a StackOverflow user not to lead with `* text=auto`:

> We had a repo with * text=auto, and Git guessed wrong for an image file that it was a text file, causing it to corrupt it as it replaced CR + LF bytes with LF bytes in the object database. That was not a fun one to debug.

https://stackoverflow.com/questions/21472971/what-is-the-purpose-of-text-auto-in-gitattributes-file